ETC_en.tsv

```diff
@@ -5128,31 +5128,31 @@ ETC_20150317_005127	My work shift ends soon!
 ETC_20150317_005128	Please defeat Stone Froster and obtain Nuclear Frost!
 ETC_20150317_005129	What?
 ETC_20150317_005130	What is that sound?
-ETC_20150317_005131	Please defeat Gargoyle!
-ETC_20150317_005132	Please defeat Devilglove!
+ETC_20150317_005131	Please defeat the Gargoyle!
+ETC_20150317_005132	Please defeat the Demon Glove!
 ETC_20150317_005133	Northern Guards
 ETC_20150317_005134	Pile of Books
 ETC_20150317_005135	First mobile device
 ETC_20150317_005136	Second mobile device
-ETC_20150317_005137	Damn it. It was a trap. This is a restrainment spell.
-ETC_20150317_005138	Flam Fusion Machine
+ETC_20150317_005137	Damn it. It was a trap. This is a Restrainment Spell.
+ETC_20150317_005138	Flame Fusion Machine
 ETC_20150317_005139	Have a good time
 ETC_20150317_005140	Magic regulating valve
 ETC_20150317_005141	Scattered books
 ETC_20150317_005142	Scattered books
 ETC_20150317_005143	Hahahahaha....
-ETC_20150317_005144	Chase Grita who took the Essence of Fire!
+ETC_20150317_005144	Chase the Grita who took the Essence of Fire!
 ETC_20150317_005145	Well then.
 ETC_20150317_005146	Magic Circle
-ETC_20150317_005147	Helga Cercle
+ETC_20150317_005147	Helga Circle
```

Comment:
Helga cercle seems to be a boss name, but spelled differently in files : 
ITEM_20150317_002047	Helgacercle Card
asked question here :https://github.com/Treeofsavior/EnglishTranslation/issues/308